pom.xml

```diff
@@ -142,7 +142,7 @@
         </site>
         <snapshotRepository>
             <id>release_artifacts_gradoop</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots/org/gradoop</url>
```

Comment:
This was a critical issue ... I added that suffix somewhen in 2022 but do not know why.